src/main/java/org/folio/inventory/dataimport/entities/OlItemAccumulativeResults.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.inventory.dataimport.entities;
+
+import io.vertx.core.json.JsonObject;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Interface for storing intermediate results of the Item(errors and successful) Handler's processing between runs if Optimistic Locking reveals.
+ * It is needed for provided intermediate results between runs via DataImportEventPayload  by key "OL_ACCUMULATIVE_RESULTS".
+ * It avoids using class-level fields.
+ */
+@Getter
+@Setter
+public class OlItemAccumulativeResults{
```

Comment:
it seems that it's not necessary to have different classes for  holdings and items since they operate JsonObjects and PartialErrors

Reply:
@tatsiana-tarhonskaya For Items there will be JsonObject, but for Holdings - it will be HoldingsRecord. There are some complexities for mapping Item->Json and Json->Item, that's why it was changed there from "Item"-type to the "JsonObject"-one.

Reply:
@tatsiana-tarhonskaya "complexity" - I mean that there should be extra specific-field manipulation for the Items.

Reply:
@VRohach , yes, missed the "HordingsRecord" part